pkgs/development/ocaml-modules/owee/default.nix

```diff
@@ -1,15 +1,17 @@
-{ lib, buildDunePackage, fetchFromGitHub }:
+{ lib, buildDunePackage, fetchFromGitHub, pkg-config }:
 
 buildDunePackage rec {
   minimumOCamlVersion = "4.06";
   pname = "owee";
-  version = "0.3";
+  version = "0.4";
+
+  nativeBuildInputs = [ pkg-config ];
 
   src = fetchFromGitHub {
```

Comment:
The source is there: https://github.com/let-def/owee/releases/download/v0.4/owee-0.4.tbz

Reply:
I don't understand this comment, do you want me to change how the package is fetched?

Reply:
Okay, I didn't touch how we get the package since I just bumped the sources. If it's better for some reason to get the manually uploaded tarball I can obviously change it. My guess is that the uploaded file is created by dune-release and should be basically the same as the repo.

Reply:
I have now changed how we get `owee` sources, are we good to go on this now?